src/components/Reports/ReportsBox/ReportsBox.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { InjectAppServices } from '../../../services/pure-di';
+import { FormattedMessage } from 'react-intl';
+import Loading from '../../Loading/Loading';
+import './ReportsBox.css';
+
+class ReportsBox extends React.Component {
+  constructor({ dependencies: { datahubClient } }) {
+    super();
+
+    /** @type { import('../../services/datahub-client').DatahubClient } */
+    this.datahubClient = datahubClient;
+
+    this.state = {
+      isLoading: true,
+      dateTo: new Date(),
+    };
+
+    this.getVisitsByPeriod = this.getVisitsByPeriod.bind(this);
+  }
+
+  async getVisitsByPeriod(domainName, periodSelectedDays) {
+    let dateFrom = new Date();
+    dateFrom.setDate(dateFrom.getDate() - parseInt(periodSelectedDays));
+
+    const visits = await this.datahubClient.getVisitsByPeriod(domainName, dateFrom);
+    this.setState({
+      visits: visits,
+      dateFrom: new Date(dateFrom),
```

Comment:
Should you also update `dateTo`?

Reply:
the day of today never change. or i missing something?

Reply:
Imagine that you are at midnight: the day could change. 

Also, if I am not wrong, you are using complete dates (with time, I mean, you are not cleaning it), so, eventually, we would decide to show also the time.

Summarizing, visits has the count of visits between _dateFrom_ and the time of the request (in place of the first rendering _dateTo_).

Does it make sense?

Reply:
yes, changed it